app/models/miq_queue_worker_base/runner.rb

```diff
@@ -35,7 +35,7 @@ def thresholds_exceeded?
   def get_message_via_drb
     loop do
       begin
-        msg_id, lock_version = @worker_monitor_drb.get_queue_message(@worker.pid)
+        msg_id, lock_version = worker_monitor_drb.get_queue_message(@worker.pid)
```

Comment:
We already have a `attr_reader` on the `@worker_monitor_drb` so i'm using it here so it's easier to test.